### PR TITLE
components moved out of Rx and react path working with metalsmith build.

### PR DESCRIPTION
**Pull request by crwallace**
Rx react can now be found at `/rx`

### CODE REVIEW

File: content/includes/footer.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
+<a class="sr-only" href="#feedback-tool">Give feedback on this page</a>
+
+
```
Comment (awong-dev) [inline content/includes/footer.html]: remove unneeded newlines?

File: content/includes/header.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+<!--[if lt IE 9]>
+<link rel='stylesheet' href='/assets/css/ie.css'>
+<![endif]-->
+
+<!-- We participate in the US government's analytics program. See the data at analytics.usa.gov. -->
+<script src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=VA" id="_fed_an_ua_tag"></script>
+
+
+
+<!--{% if site.production %}
```
Comment (awong-dev) [inline content/includes/header.html]: Should this chunk just be removed rather than commented out?

File: package.json [hunk 3]
```diff
@@ -46,6 +46,8 @@
     "metalsmith": "^2.1.0",
     "metalsmith-collections": "^0.7.0",
     "metalsmith-concat-convention": "^1.0.0",
+    "metalsmith-date-in-filename": "0.0.14",
+    "metalsmith-filenames": "https://registry.npmjs.org/metalsmith-filenames/-/metalsmith-filenames-1.0.0.tgz",
```
Comment (awong-dev) [inline package.json]: this is weird... why is everything else a number and this a URL?

File: package.json [hunk 4]
```diff
@@ -46,6 +46,8 @@
     "metalsmith": "^2.1.0",
     "metalsmith-collections": "^0.7.0",
     "metalsmith-concat-convention": "^1.0.0",
+    "metalsmith-date-in-filename": "0.0.14",
```
Comment (awong-dev) [inline package.json]: Does this need to be an exact version number?  I was expecting a ^ before the version number.

File: package.json (hunk 3 above)
Comment (awong-dev) [inline package.json]: Am also curious if this was added via `npm install -D metalsmith-date-in-filename` or manually.  I've think I've seen this once before but it's an old memory. npm confuses me. :-/

**Comment by awong-dev** [comment]
@crwallace Can you also update the shirnkwrap to ensure all the package version are locked?

`npm shrinkwrap --dev`

File: package.json (hunk 3 above)
Comment (crwallace) [inline package.json]: This was added via `npm install -D metalsmith-date-in-filename`

File: content/includes/header.html (hunk 2 above)
Comment (crwallace) [inline content/includes/header.html]: I am not done jekyll-ifying this but I wanted to get this up and running for the Rx team and this doesn't affect them.

File: content/includes/footer.html (hunk 1 above)
Comment (crwallace) [inline content/includes/footer.html]: ok

File: package.json (hunk 4 above)
Comment (crwallace) [inline package.json]: ok

File: package.json (hunk 3 above)
Comment (crwallace) [inline package.json]: Just tried to re-install it twice. still a url.

Comment (awong-dev) [inline package.json]: Crazy. I just did it on my side and it's `    "metalsmith-filenames": "^1.0.0",`

Can we manually switch it to a the version number?

I've been reading about what the url means in package.json and I'm pretty sure it's not what we want.

File: package.json (hunk 4 above)
Comment (awong-dev) [inline package.json]: weird. okay.

**Comment by awong-dev** [comment]
LGTM

**Comment by awong-dev** [comment]
Travis is broken in a weird way so merging preemptively.

File: package.json (hunk 3 above)
Comment (crwallace) [inline package.json]: sure!

**Pull request merged**